src/core/identifiers/identifiers.ts

```diff
@@ -1,20 +1,20 @@
 import { BigNumber } from "ethers";
 
 /**
- * DSNPMessageId represents a DSNP message id following the DSNP
+ * dsnpAnnouncementId represents a DSNP Announcement Id following the DSNP
```

Comment:
Why did we downcase this type? I thought we were uppercasing all types as a rule.

Reply:
Mistake. Will Fix.